docs/source/getting-started/tutorial.rst

```diff
@@ -31,17 +31,16 @@ and the commands to run:
 
   setup: |
     set -e  # Exit if any command failed.
-    conda install pytorch==1.12.1 cudatoolkit=11.3 -c pytorch -y
     git clone https://github.com/huggingface/transformers/ || true
     cd transformers
-    pip3 install .
+    pip install .
     cd examples/pytorch/text-classification
-    pip3 install -r requirements.txt
+    pip install -r requirements.txt torch==1.12.1+cu113 --extra-index-url https://download.pytorch.org/whl/cu113
```

Comment:
Question: Wondering if it works having `conda install` after the `pip install -r requirements.txt`, and why we fall back to the `pip` version

Reply:
Tried that out: it worked on GCP but not on AWS.